debian/changelog

```diff
@@ -1,3 +1,97 @@
+  * cli: snap command migrated to docopts.
+  * Enabling testing with new cli.
+  * Fix ordering for conditional negations in catkin tests.
+  * Replace use of `assert_not_called` in the catkin tests.
+  * General documentation update.
+  * Add licensing support. (LP: #1527453)
+  * snap.yaml implementation with support for readme.md and package.yaml.
+    (LP: #1532842)
+  * Documentation refresh. (LP: #1533021)
+  * Support the new security-override semantics. (LP: #1533293)
+  * Remove ssh module from snapcraft. (LP: #1533400)
+  * Fix environment variable handling. (LP: #1531481)
+  * Wrap apps with the app name. (LP: #1534150)
+  * Replace SNAP envvars with the 16.04 ones. (LP: #1533552)
+  * Add bin paths to the to the shebang replace paths in repo (LP: #1534812)
+  * Initialize CMAKE_PREFIX_PATH to [] for ros (LP: #1535309)
+
+  [ Leo Arias ]
+  * Added a test to check that the snapcraft files are not copied to the build dir.
+  * On autopkgtests, run the integration tests, build the examples but do not
+    install them, do not run the unittests.
+  * Translated the plainbox tests to python.
+  * Workaround the failure of the binaries in trusty.
+  * Updated the runtests script error message.
+  * Updated the autopkg integration tests.
+  * Handle the filter parameter for examples tests. (LP: #1533393)
+  * Split the static and unit tests. (LP: #1533397)
+  * Added the option to output subunit results in examples tests. (LP: #1533403)
+  * Updated the tomcat example network capability. (LP: #1533736)
+  * Added the network-listener capability. (LP: #1534784)
+  * Add a security-override for chown.
+
+  [ Kyle Fazzari ]
+  * Catkin plugin: Add support for ROS tools.
+  * Add contribution guide.
+  * Resolve FIXME in Catkin plugin.
+  * Add a tutorial for turning a ROS project into a .snap.
+  * Catkin plugin: use rosdep for dependency resolution.
+  * Python{2,3} plugins: Make site-packages link relative. (LP: #1523384)
+  * Catkin plugin: Refactor build.
+  * ROS documentation: Fix missing step to make script executable.
+  * Catkin plugin: Improve cmake path rewrite.
+  * Add support for mesa libraries. (LP: #1531620)
+  * Autotools plugin: Set autogen.sh executable. (LP: #1530995)
+  * Update CONTRIBUTING.md to require a bug. (LP: #1532195)
+  * Ensure rosdep resolves dependencies using Trusty. (LP: #1532241)
+  * Catkin plugin: Use in-snap python instead of OS-provided. (LP: #1533297)
+  * Always migrate files, even if they already exist. (LP: #1534800)
+
+  [ Alexander Sack ]
+  * Use archive.apache.org URL to ensure we refer to a stable URL that does
+    not disappear on latest release.
+
+  [ Jonathan Cave ]
+  * Go plugin creating invalid env. (LP: #1524374)
+
+  [ Renat Galimov ]
+  * Close not called on the qml plugin. (LP: #1531994)
+
+  [ Vamshi Balanaga ]
+  * Added test for git sources in sources.py (LP: #1534411)
+
+ -- Leonardo Arias Fonseca <leo.arias@canonical.com>  Fri, 15 Jan 2016 17:52:01 -0600
```

Comment:
In theory this is why we need to craft a changelog, create a packaging branch and keep CHANGELOG updated with commits on master and debian/changelog abstained to packaging bugs (this is the way it is originally intended).